air/src/lib.rs

```diff
@@ -277,3 +277,12 @@ impl Serializable for PublicInputs {
         self.stack_outputs.write_into(target);
     }
 }
+
+impl vm_core::ToElements<Felt> for PublicInputs {
```

Comment:
nit: we don't use this syntax often (i.e. full import path on use). We usually import everything at BOF, and then use the imported items.

Reply:
We actually have another `ToElements` trait defined in `miden-core` - so, might be a good issue to figure out how to de-duplicate this.